apps/advisor/pkg/app/checkscheduler/checkscheduler.go

```diff
@@ -227,7 +229,9 @@ func (r *Runner) cleanupChecks(ctx context.Context, logger logging.Logger) error
 	}
 
 	for _, checks := range checksByType {
+		logger.Debug("Checking checks", "checkType", checks[0].GetStaticMetadata().Identifier(), "numChecks", len(checks))
```

Comment:
i am assuming the answer is yes, but just to double check, will there always be at least one item in the checks slice?

Reply:
I'm going to remove the changes of this file and open a new PR for this.